x-pack/plugins/cases/public/components/add_comment/index.test.tsx

```diff
@@ -52,6 +52,10 @@ describe('AddComment ', () => {
   beforeEach(() => {
     jest.clearAllMocks();
     useCreateAttachmentsMock.mockImplementation(() => defaultResponse);
+    Object.defineProperty(window, 'sessionStorage', {
```

Comment:
There is no need to mock the `sessionStorage`. You can use it in `jest`. You can add items and get items and see if your code behaves correctly.

Reply:
removed mocks